validator/OWNERS

```diff
@@ -4,7 +4,7 @@
 {
   rules: [
     {
-      owners: [{name: 'ampproject/wg-caching', notify: true}],
+      owners: [{name: 'ampproject/wg-caching', require: true}],
```

Comment:
```suggestion
      owners: [{name: 'ampproject/wg-caching', required: true}],
```

![image](https://user-images.githubusercontent.com/6694512/95782849-9939cc00-0c9e-11eb-9461-cd57b4b7c5ca.png)

Reply:
Update: I committed this suggested change directly though the UI to save you one round-trip checking the thread, updating, and waiting for presubmits